client/src/App.js

```diff
@@ -0,0 +1,17 @@
+import logo from './logo.svg';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/NamamiShanker/twitterSOS/pullRequest?prid=7540437)
```suggestion
import logo from "./logo.svg";
```